user_docs/en/changes.t2t

```diff
@@ -8,7 +8,7 @@ What's New in NVDA
 
 == New Features ==
 - Added support for Bluetooth Low Energy HID Braille displays. (#15470)
-- The Add-on Store now supports installing add-ons in bulk by selecting multiple add-ons. (#15350)
+- The Add-on Store now supports actions (installing, enabling add-ons, etc.) in bulk by selecting multiple add-ons. (#15350, #15623, @CyrilleB79)

```

Comment:
```suggestion
- The Add-on Store now supports bulk actions (e.g. installing, enabling add-ons) by selecting multiple add-ons. (#15350, #15623, @CyrilleB79)
```

Reply:
>I have modified the change log item since batch install was available but I have implemented all the other actions. I have also added my GitHub's name. Unfortunately, it gives the impression that I have made all the job, whereas @seanbudd / NV Access has provided the initial work in https://github.com/nvaccess/nvda/pull/15350. Is it worth thinking to a way to show that an item has been contributed both by NV Access and an external contributor? Or by two external contributors? This would need to be clarified.

I wouldn't worry about this, we don't need to be credited.

Reply:
In the current PR, that's not so much a problem since I understand the initial PR (#15350). I was just mentioning since in an hypothetic future case, I (or another external contributor) may be associated this way with a PR that I do not fully understand or for which I have not the background.